### PR TITLE
(#165026726):Create services model

### PR DESCRIPTION
**Pull request by KicaRonaldOkello**
### What does this PR do?
This PR creates a model for services.

#### Description of Task to be completed?
- [x] Create a model class for services.
- [x] Run migratons for the model
#### How should this be manually tested?

- Checkout to the branch `git checkout ch-create-user-modal-165026611`

- Make migrations `python manage.py makemigrations`

- Then migrate `python manage.py migrate`

#### What are the relevant pivotal tracker stories?
[#165026726](https://www.pivotaltracker.com/n/projects/2223015/stories/165026726)

### CODE REVIEW

**Comment by kimbugp** [comment]
@KicaRonaldOkello , please rebase with develop and recreate a new migration file to avoid conflicting conflicts

**Comment by kimbugp** [comment]
@KicaRonaldOkello , could you add tests for the model

File: apps/services/models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.template.defaultfilters import slugify
+from django.contrib.auth.models import User
+
+class Services(models.Model):
+    name = models.CharField(max_length=255)
```
Comment (kimbugp) [inline apps/services/models.py]: @KicaRonaldOkello , I think services shouldnot be tied to a stylist.
Services should be open for all stylists to add to there arsenal.

Comment (kimbugp) [reply apps/services/models.py]: I think services should be like categories

**Review by kimbugp — CHANGES_REQUESTED** [review]

File: manage.py [hunk 2]
```diff
@@ -12,4 +12,8 @@
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
         ) from exc
+    if 'runserver' in sys.argv:
```
Comment (kimbugp) [inline manage.py]: @KicaRonaldOkello , this should be removed. It has a security risk

**Review by kimbugp — CHANGES_REQUESTED** [review]

File: manage.py (hunk 2 above)
Comment (KicaRonaldOkello) [reply manage.py]: will remove it

Comment (KicaRonaldOkello) [reply manage.py]: will remove it

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kimbugp) [inline .vscode/settings.json]: @KicaRonaldOkello , add this in the gitignore

File: apps/services/models.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    def __str__(self):
+        return self.name
+
+class HairStyle(BaseModel):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    picture = models.URLField()
+    slug = models.SlugField()
+    price_range = models.DecimalField(max_digits=8, decimal_places=0)
```
Comment (kimbugp) [inline apps/services/models.py]: @KicaRonaldOkello , i think the price range shouldnot be limited to decimals.
A user should be able to provide something like `50000-80000`

**Review by kimbugp — CHANGES_REQUESTED** [review]

**Review by kimbugp — APPROVED** [review]

**Pull request merged**